### ISSUE TITLE
Missing packages in installation

### ISSUE THREAD

**Issue by gschivley** [NREL/OpenOA#92]
It looks like h5pyd, which is imported in operational_analysis/methods/quality_check_automation.py, isn't in setup.py or requirements.txt. 

Since the examples are provided as jupyter notebooks and use matplotlib to show results, I'd also suggest that those packages should be either included in the default installation or called out as needed for the examples. matplotlib is included in the requirements.txt - which is referenced under [development](https://openoa.readthedocs.io/en/latest/install.html#development) installation - but jupyter isn't included anywhere. These are less critical than h5pyd though.

Part of review (openjournals/joss-reviews#2171)

**Comment by jordanperr** [comment]
Thanks for catching these issues.

In branch joss-review:

- h5py has been added to setup.py and requirements.txt
- matplotlib is now included in requirements.txt and setup.py, since it is required to run core functionality of OpenOA.
- We consider jupyter like an IDE or text editor that a developer may choose to view the example notebooks. The notebooks can be opened using other tools (e.g., VSCode, Github's web interface, pandoc). I have added a line in the Readme suggesting Jupyter to open the example notebooks.
